apps/tmux/tmux.py

```diff
@@ -19,7 +19,7 @@
 class TmuxActions:
     def tmux_prefix():
         """press control and the configured tmux prefix key"""
-        actions.key(f"{setting_tmux_prefix_key.get()}")
+        actions.key(f"{settings.get('user.tmux_prefix_key')}")
```

Comment:
Think this may have been generated due to .talon → .py action conversion; would just get rid of the format string.